client/gutenberg/extensions/publicize/connection.jsx

```diff
@@ -56,6 +69,32 @@ class PublicizeConnection extends Component {
 			</Notice>
 		);
 
+	/**
+	 * Displays a message when a connection requires reauthentication. We used this when migrating LinkedIn API usage from v1 to v2,
+	 * since the prevous OAuth1 tokens were incompatible with OAuth2.
+	 *
+	 * @returns {object|null} Notice about reauthentication
+	 */
+	maybeDisplayLinkedInNotice = () =>
+		this.connectionNeedsReauth() && (
+			<Notice className="jetpack-publicize-notice" isDismissible={ false } status="error">
+				<p>
+					{ __(
+						'Your LinkedIn connection needs to be reauthenticated to continue working – head to Sharing to take care of it.'
```

Comment:
Wanted to share that "head to Sharing" seems a bit imperative, as if we're ordering the user 😉 Could we make it a bit more friendly?

Reply:
This has already been reviewed by editorial.